bump_rpm_packaging

```diff
@@ -12,20 +12,19 @@ git checkout -b rpm/release-$FULLVERSION $PACKAGING_GIT_REMOTE/$REMOTE_BRANCH
 for package in $RPM_PACKAGES ; do
 	(
 		cd packages/foreman/$package
+		spec_file="${package}.spec"
 		if [[ $FULLVERSION == *-RC* ]] ; then
 			if [[ FULLVERSION == *-RC1 ]] ; then
 				echo "TODO: bump release"
 			fi
-			sed -i 's/^\(Version:\s\+\).\+$/\1'${FULLVERSION%%-RC*}'/ ; s/^\(%global prerelease\s\+\).\+$/\1'${FULLVERSION##*-}'/' $package.spec
+			sed -i 's/^\(Version:\s\+\).\+$/\1'${FULLVERSION%%-RC*}'/ ; s/^\(%global prerelease\s\+\).\+$/\1'${FULLVERSION##*-}'/' $spec_file
 		else
-			sed -i 's/^\(Version:\s\+\).\+$/\1'$FULLVERSION'/ ; s/^\(%global release\s\+\).\+$/\11/ ; /^\(%global prerelease\s\)/d' $package.spec
-		fi
-		git add $package.spec
-		if [[ $package != foreman-release ]] ; then
-			git rm --ignore-unmatch --force *.tar.bz2
-			spectool -g $package.spec
-			git annex add *.tar.bz2
+			sed -i 's/^\(Version:\s\+\).\+$/\1'$FULLVERSION'/ ; s/^\(%global release\s\+\).\+$/\11/ ; /^\(%global prerelease\s\)/d' $spec_file
 		fi
+		git add $spec_file
+		spectool --list-files $spec_file | cut -d' ' -f2 | grep http | xargs --no-run-if-empty -n 1 basename | xargs --no-run-if-empty git rm
```

Comment:
this needs to happen *before* you change the version in the spec file, as otherwise it will `git rm` an non-existing file ;)